gnomad_constraint/utils/constraint.py

```diff
@@ -479,3 +479,45 @@ def compute_constraint_metrics(
             all_ht = all_ht.annotate(**ht[all_ht.key])
 
     return all_ht
+
+
+def split_context_ht(
```

Comment:
I think this is only supposed to return one Table so I changed it to do so in my suggestions